src/txmempool.h

```diff
@@ -129,6 +129,7 @@ class CTxMemPoolEntry
     int64_t GetSigOpCostWithAncestors() const { return nSigOpCostWithAncestors; }
 
     mutable size_t vTxHashesIdx; //!< Index in mempool's vTxHashes
+    mutable uint64_t m_epoch; //!< epoch when last touched, useful for graph algorithms
```

Comment:
2ccb7cc

I think you considered it but why not m_entry_epoch, to avoid names collision for reviewers?

Reply:
fair point -- I felt that prefixing it with the class name was a bit gauche compared to just naming it the same thing. It's also an internal detail that isn't going to be referenced or inspected by any callers, so it isn't going to yield much name colliding code.

I suggest that when @ajtowns refactors a lock analyzer friendly interface he can pick whatever variable names he thinks appropriate (except maybe m_aj_is_number_one).